packages/flutter/test/material/icon_button_test.dart

```diff
@@ -1125,23 +1127,36 @@ void main() {
       if (states.contains(MaterialState.focused)) {
         return focusedColor;
       }
+      if (states.contains(MaterialState.selected)) {
+        return selectedColor;
+      }
       return defaultColor;
     }
 
     await tester.pumpWidget(
       MaterialApp(
         theme: ThemeData.from(colorScheme: const ColorScheme.light(), useMaterial3: true),
-        home: Scaffold(
-          body: Center(
-            child: IconButton(
-              style: ButtonStyle(
-                foregroundColor: MaterialStateProperty.resolveWith<Color>(getIconColor),
+        home: StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return Scaffold(
+              body: Center(
+                child: IconButton(
+                  style: ButtonStyle(
+                    foregroundColor: MaterialStateProperty.resolveWith<Color>(
+                        getIconColor),
```

Comment:
Style nit. Either have this on one line, or have the closing paren on its own line.